core/src/utils/account_id20.rs

```diff
@@ -0,0 +1,162 @@
+// Copyright 2019-2024 Parity Technologies (UK) Ltd.
+// This file is dual-licensed as Apache-2.0 or GPL-3.0.
+// see LICENSE for license details.
+
+//! The "default" Substrate/Polkadot AccountId. This is used in codegen, as well as signing related bits.
```

Comment:
I have troubles to understand `The "default" Substrate/Polkadot AccountId` part, to my understanding AccountId32 is the default one and AccountId20 is used for chains that are using ethereum accounts.

I would rather say that it's used for chains that is using ethereum accounts which are different from the default AccountId32 that is used by substrate by default.

For substrate-based chains that exposes AccountId20 in the metadata, this type can used to generate code when the correct AccountId rather than converting back and forth between AccountId20 and AccountId32

That is my understanding of it...

Reply:
good catch, i didnt read into the top part of the file when copying license for accountId32 and brought this with it. replaced this part